src/main/java/com/dani/blog/domain/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.dani.blog.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * @author dani
+ */
+@NoArgsConstructor
+@Getter
+@Setter
+public class Comment {
```

Comment:
I see that this is the DTO and the CommentEntity is the domain entity. Even though this is matter of convention and might differ based on approach, I find this naming confusing. I would use Comment either for the interface or the domain entity, but would never use it for the Dto.